tests/test_server.py

```diff
@@ -190,8 +190,11 @@ def send_request() -> None:
             formbot_data = dict(
                 domain="examples/formbot/domain.yml",
                 config="examples/formbot/config.yml",
-                stories="examples/formbot/data/stories.md",
-                nlu="examples/formbot/data/nlu.md",
+                training_files=[
```

Comment:
@chkoss btw i committed this because that's how passing training data looks like for the `train` function now - but dependent on how you implement this for the `/model/train` endpoint this might have to be changed again